Uppsala/UppEt7.xml

```diff
@@ -0,0 +1,336 @@
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl> 
+                                            <ptr target="bm:Loefgren1974Katalog"/>
+                                            <citedRange unit="page">105–106</citedRange>
+                                            <citedRange unit="number">33</citedRange>
+                                        </bibl>
+                                        <bibl> 
+                                            <ptr target="bm:Zettersteen1899Upsala"/>
+                                            <citedRange unit="page">514–515</citedRange>
+                                            <citedRange unit="number">7</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                    
+                    
+                </msDesc>
+                
+                <bibl> 
+                    <ptr target="bm:Loefgren1974Katalog"/>
+                    <citedRange unit="page">105–106</citedRange>
+                    <citedRange unit="number">33</citedRange>
+                </bibl>
+                <bibl> 
+                    <ptr target="bm:Zettersteen1899Upsala"/>
+                    <citedRange unit="page">514–515</citedRange>
+                    <citedRange unit="number">7</citedRange>
+                </bibl>
+                <bibl>
+                    <ptr target="bm:Uhlig1988Palaographie"/>
+                    <citedRange unit="page">727–728</citedRange>
+                </bibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+                <xi:fallback>
+                    <p>Definitions of prefixes used.</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage><language ident="en">English</language></langUsage>
+            <langUsage><language ident="gez">Gəʿəz</language></langUsage>
+            <langUsage><language ident="gez-tr">Gəʿəz</language></langUsage>
+            <langUsage><language ident="de">German</language></langUsage>
+            <langUsage><language ident="sv">Swedish</language></langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-07-12">Created record</change>
+            <change who="JK" when="2024-07-16">Added physical description</change>
+            <change who="JK" when="2024-07-16">Added bibliography and place name</change>
```

Comment:
I think it is enough to have one change record for all the changes within short time.